src/Connection.cpp

```diff
@@ -23,7 +23,7 @@ Connection::Connection(const std::string &uri)
   // "radio://*/80/2M/*" -> broadcast/P2P sniffing on channel 80
   // "radiobroadcast://0/80/2M -> broadcast to all crazyflies on channel 80
 
-  const std::regex uri_regex("(usb:\\/\\/(\\d+)|radio(broadcast)*:\\/\\/(\\d+|\\*)\\/(\\d+)\\/(250K|1M|2M)(\\/([a-fA-F0-9]+|\\*)(\\?[\\w=&]+)?)*)");
+  const std::regex uri_regex("(usb:\\/\\/(\\d+)|radio:\\/\\/(\\d+|\\*)\\/(\\d+)\\/(250K|1M|2M)\\/([a-fA-F0-9]+|\\*)(\\?[\\w=&]+)?)|(radiobroadcast:\\/\\/(\\d+|\\*)\\/(\\d+)\\/(250K|1M|2M))");
```

Comment:
Awesome! Just create another review request once you are ready. I think this is pretty close to being ready to be merged.